.releaserc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "branches": ["feature/semver"],
```

Comment:
klopt dit? of is dit nog van testen?

Reply:
Foutje, kan jij dat naar 'main' omzetten?